### DISCUSSION TITLE
Connecting Zettels

### DISCUSSION THREAD

**Discussion by DeLub** [bramses/bramses-highly-opinionated-vault-2023#8] [General]
Hi,

Really like your vault. I took over quite some ideas and use them in my vault now as well.

Question, though: how do you create links between Zettels? Luhmann positioned the Zettels one after another and thus created links. In addition he added cross links. I lack the experience to judge what is the right approach. In your experience, do you only use manual cross links? Maybe in addition with MOCs? Or do you use another method as well, perhaps replicating a tree like structure as Luhmann did?

Kind regards,
Alex

**Comment by bramses** [comment]
Great question! I create manual links in the style of Zettel from Zettelkasten.de (https://zettelkasten.de/posts/search-alone-is-not-enough/) and thanks to the Luhmann plugin (pre installed in BHOV) I added a hot key to insert a Zettel that pops up a modal and searches all valid Zettels that follow the ID format and inserts the link at a cursor.

Just make sure to provide context!

Edit: an example of the unique Id formatting I also use (https://zettelkasten.de/introduction/#the-unique-identifier)
